### ISSUE TITLE
[bayesian-priors] Documentation (Trac #1129)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1123]
The translation from dox to rst wasn't entirely complete, so this needs to be cleaned up.

There's a comment in the code (I3TableZenithWeight.cxx):
// numbers of default table are taken from like_ama_zenwght & zenith_weight
// Ask Gary Hill.
// siegmund/recoos/reco_amanda.c, line 133-146.

Can we get better documentation for this.  If I searched around a bit I might be able to find that siegmund code, but most people can't and shouldn't have to.  How about we ask Gary?

Same with I3PowExpZenithWeight.cxx.  Comments say to ask Paolo and Gary.  Good thing they're still on IceCube.

Migrated from https://code.icecube.wisc.edu/ticket/1129
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "The translation from dox to rst wasn't entirely complete, so this needs to be cleaned up.\n\nThere's a comment in the code (I3TableZenithWeight.cxx):\n// numbers of default table are taken from like_ama_zenwght & zenith_weight\n// Ask Gary Hill.\n// siegmund/recoos/reco_amanda.c, line 133-146.\n\nCan we get better documentation for this.  If I searched around a bit I might be able to find that siegmund code, but most people can't and shouldn't have to.  How about we ask Gary?\n\nSame with I3PowExpZenithWeight.cxx.  Comments say to ask Paolo and Gary.  Good thing they're still on IceCube.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[bayesian-priors] Documentation", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-17T17:50:21", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T16:31:47: 
* olivas changed owner from "claudio.kopper" to "olivas"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150908T19:48:18: olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151014T12:55:19: kjmeagher changed owner from "olivas" to "kjmeagher"

**Comment by icecube-trac** [comment]
Trac update at 20151029T19:27:54: 
* kjmeagher changed status from "accepted" to "closed"
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 138954:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138954"
> after consulting with Paolo, I updated the comments to reflect the source of the constants in the code and added a link fixes #1123
> Also considerable improvment to the documentation
> also remove useless README
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
